src/documentation/reference/introduction/XDR.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: XDR
+order: 6
+---
+
+In the Stellar network, transactions are encoded using a standardized protocol called [External Data Representation](https://en.wikipedia.org/wiki/External_Data_Representation) (XDR).
+
+In Horizon, you will only encounter XDR when [posting](../resources/transactions/post-transaction.mdx) and [getting](../resources/transactions/retrieve-transaction.mdx) transactions and in the [ledger](../resources/ledgers/retrieve-ledger.mdx) header.
+
+When you post a transaction, a client will encode the transaction as XDR before submitting it to Horizon.
+
+When you request a transaction, Horizon returns some data about the transaction in human-readable JSON. The full canonical data about the transaction is encoded in machine-readable XDR, available in XDR attributes at the end of the response.
+
+You can decode this XDR in the Stellar Laboratory’s [XDR Viewer](https://www.stellar.org/laboratory#xdr-viewer).
+
+<NestedTable>
+
+- ATTRIBUTE
+  - TYPE
+  - DESCRIPTION
+- envelope_xdr
+  - string
+  - The XDR encoded transaction as stellar-core sees it.
+- result_xdr
+  - string
+  - The effects of a transaction encoded in XDR.
+- result_meta_xdr
+  - string
+  - The details about the effects of a transaction encoded in XDR.
+- fee_meta_xdr
+  - string
+  - The fees associated with the transaction encoded in XDR.
+
+</NestedTable>
```

Comment:
it's important to leave a paragraph break before and after content inside a component so that the content will render on github's site.